.jenkins/Jenkinsfile

```diff
@@ -0,0 +1,117 @@
+// Load Jenkins shared library
+jenkinsBranch = 'sift-python'
+sharedLib = library("shared-lib@${jenkinsBranch}")
+
+def siftPythonWorkflow = sharedLib.com.sift.ci.SiftPythonWorkflow.new()
+def ciUtil = sharedLib.com.sift.ci.CIUtil.new()
+def stackdriver = sharedLib.com.sift.ci.StackDriverMetrics.new()
+
+// Default GitHub status context for automatically triggered builds
+def defaultStatusContext = 'Jenkins:auto'
+
+// Pod template file for Jenkins agent pod
+// Pod template yaml file is defined in https://github.com/SiftScience/jenkins/tree/master/resources/jenkins-k8s-pod-templates
+def python2PodTemplateFile = 'python-2-7-pod-template.yaml'
+def python3PodTemplateFile = 'python-3-10-pod-template.yaml'
+def python2PodLabel = "python2-${BUILD_TAG}"
+def python3PodLabel = "python3-${BUILD_TAG}"
+
+
+// GitHub repo name
+def repoName = 'sift-python'
+
+pipeline {
+    agent none
+    options {
+        timestamps()
+        skipDefaultCheckout()
+        disableConcurrentBuilds()
```

Comment:
Please add `disableRestartFromStage()` option.  I am adding this to `code` repo as well.